.github/workflows/smoketest-chart.yaml

```diff
@@ -0,0 +1,77 @@
+name: Helm Chart Smoketest
+
+on: [push, workflow_dispatch, pull_request]
+
+env:
+  K8S_VERSION: v1.32.3
+  MICROK8S_CHANNEL: 1.32/stable
+
+jobs:
+  helm-install-smoke-test:
+    permissions: {}
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        config:
+          - { type: "kind" }
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Install Helm
+        uses: Azure/setup-helm@v4
+        with:
+          version: v3.18.0
+      # Note: 'uses' doesn't support variable interpolation, hence the
+      # k8s-specific steps below.
+      # Ref: https://github.com/orgs/community/discussions/25824
+      - name: Create kind cluster
+        if: matrix.config.type == 'kind'
+        uses: helm/kind-action@a1b0e391336a6ee6713a0583f8c6240d70863de3 # v1.12.0
+        with:
+          cluster_name: kind
+          version: v0.29.0
+          node_image: kindest/node:${{ env.K8S_VERSION }}
+      - name: Helm install OpenBao
+        run: |
+          helm install openbao \
+            --namespace openbao \
+            --create-namespace \
+            --debug \
+            charts/openbao
+      - name: Wait until OpenBao is in Running state
+        run: |
+          timeout 60s bash -c 'until [[ "$(kubectl -n openbao get pods $(kubectl -n openbao get pods -l statefulset.kubernetes.io/pod-name=openbao-0 -ojsonpath="{.items[].metadata.name}") -o jsonpath="{.status.phase}" 2>/dev/null)" == "Running" ]]; do kubectl -n openbao get pods $(kubectl get pods -n openbao -l statefulset.kubernetes.io/pod-name=openbao-0 -ojsonpath="{.items[].metadata.name}") -o jsonpath="{.status.phase}" && sleep 5; done'
+      - name: Run OpenBao Agent
```

Comment:
This step is initializing an `OpenBao Node` and not an `OpenBao Agent` or am I missing something?

Reply:
Good catch. We are talking about an OpenBao instance :)